# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gdbm/hotfix-ssize_t.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

w/ musl:

In file included from wordwrap.c:18:
gdbmapp.h:74:30: error: unknown type name 'ssize_t'; did you mean 'size_t'?
   74 |                              ssize_t (*writer) (void *, const char *, size_t),
      |                              ^~~~~~~
      |                              size_t
make[3]: *** [Makefile:571: wordwrap.o] Error 1

--- gdbm-1.25/tools/gdbmapp.h.vanilla	2025-04-07 23:40:25.430533368 +0200
+++ gdbm-1.25/tools/gdbmapp.h	2025-04-07 23:42:02.800510448 +0200
@@ -16,6 +16,7 @@
 
 #include <stdlib.h>
 #include <stdarg.h>
+#include <sys/types.h>
 #include "gettext.h"
 #ifdef HAVE_LOCALE_H
 # include <locale.h>
